diff -up libwebp-1.1.0/cmake/cpu.cmake.omv~ libwebp-1.1.0/cmake/cpu.cmake
--- libwebp-1.1.0/cmake/cpu.cmake.omv~	2020-12-14 21:29:04.039224601 +0100
+++ libwebp-1.1.0/cmake/cpu.cmake	2020-12-14 21:29:15.750401113 +0100
@@ -39,7 +39,7 @@ else()
   set(SIMD_ENABLE_FLAGS
       "-msse4.1;-msse2;-mips32;-mdspr2;-mfpu=neon;-mmsa")
   set(SIMD_DISABLE_FLAGS
-      "-mno-sse4.1;-mno-sse2;;-mno-dspr2;;-mno-msa")
+      "-mno-sse4.1;-mno-sse2;;;;")
 endif()
 
 set(WEBP_SIMD_FILES_TO_NOT_INCLUDE)
