diff -up libwebp-1.1.0/CMakeLists.txt.omv~ libwebp-1.1.0/CMakeLists.txt
--- libwebp-1.1.0/CMakeLists.txt.omv~	2020-07-01 22:23:08.580485518 +0200
+++ libwebp-1.1.0/CMakeLists.txt	2020-07-01 22:23:23.642681057 +0200
@@ -467,7 +467,7 @@ if(WEBP_BUILD_VWEBP)
     target_link_libraries(vwebp
                           ${OPENGL_LIBRARIES}
                           exampleutil
-                          ${GLUT_glut_LIBRARY}
+                          ${GLUT_glut_LIBRARY} glut
                           imageioutil
                           webp
                           webpdemux)
diff -up libwebp-1.1.0/examples/Makefile.am.omv~ libwebp-1.1.0/examples/Makefile.am
--- libwebp-1.1.0/examples/Makefile.am.omv~	2020-07-01 22:22:02.715624319 +0200
+++ libwebp-1.1.0/examples/Makefile.am	2020-07-01 22:22:29.417974714 +0200
@@ -81,7 +81,7 @@ vwebp_LDADD  =
 vwebp_LDADD += libexample_util.la
 vwebp_LDADD += ../imageio/libimageio_util.la
 vwebp_LDADD += ../src/demux/libwebpdemux.la
-vwebp_LDADD += $(GL_LIBS)
+vwebp_LDADD += $(GL_LIBS) -lglut
 
 webpmux_SOURCES = webpmux.c
 webpmux_CPPFLAGS = $(AM_CPPFLAGS)
