diff -up libwebp-1.1.0/cmake/cpu.cmake.omv~ libwebp-1.1.0/cmake/cpu.cmake
--- libwebp-1.1.0/cmake/cpu.cmake.omv~	2020-12-14 21:29:04.039224601 +0100
+++ libwebp-1.1.0/cmake/cpu.cmake	2020-12-14 21:29:15.750401113 +0100
@@ -30,7 +30,7 @@ endfunction()
 # those are included in the names of WEBP_USE_* in c++ code.
 set(WEBP_SIMD_FLAGS "SSE41;SSE2;MIPS32;MIPS_DSP_R2;NEON;MSA")
 set(WEBP_SIMD_FILE_EXTENSIONS
-    "_sse41.c;_sse2.c;_mips32.c;_mips_dsp_r2.c;_neon.c;_msa.c")
+    "_sse41.c;;_mips32.c;_mips_dsp_r2.c;_neon.c;_msa.c")
 if(MSVC)
   # MSVC does not have a SSE4 flag but AVX support implies SSE4 support.
   set(SIMD_ENABLE_FLAGS "/arch:AVX;/arch:SSE2;;;;")
@@ -39,7 +39,7 @@ else()
   set(SIMD_ENABLE_FLAGS
       "-msse4.1;-msse2;-mips32;-mdspr2;-mfpu=neon;-mmsa")
   set(SIMD_DISABLE_FLAGS
-      "-mno-sse4.1;-mno-sse2;;-mno-dspr2;;-mno-msa")
+      "-mno-sse4.1;-msse2;;;;")
 endif()
 
 set(WEBP_SIMD_FILES_TO_NOT_INCLUDE)
